cli/src/commands/status.rs

```diff
@@ -59,10 +62,11 @@ pub(crate) fn cmd_status(
             writeln!(formatter, "The working copy is clean")?;
         } else {
             writeln!(formatter, "Working copy changes:")?;
+            let matcher = workspace_command.matcher_from_values(&args.paths)?;
```

Comment:
nit: `matcher` could be constructed before spawning pager so the error in path arguments wouldn't be paged.

Reply:
Done. Thanks for pointing that out.